lib/ansible/plugins/connection/paramiko_ssh.py

```diff
@@ -598,14 +604,14 @@ def exec_command(self, cmd, in_data=None, sudoable=True):
                     no_prompt_out += become_output
                     no_prompt_err += become_output
         except socket.timeout:
-            raise AnsibleError('ssh timed out waiting for privilege escalation.\n' + become_output)
+            raise AnsibleError('ssh timed out waiting for privilege escalation.\n' + to_native(become_output))
```

Comment:
hrm, wouldn't it just be `to_text` for anything >=3 now?

Reply:
It would be, was probably in autopilot mode when writing this.